libtransmission/peer-mgr.cc

```diff
@@ -685,9 +877,12 @@ struct tr_peerMgr
            since the blocklist has changed, erase that cached value */
         for (auto* const tor : session->torrents())
         {
-            for (auto& [socket_address, atom] : tor->swarm->pool)
+            for (auto& pool : { std::ref(tor->swarm->connectable_pool), std::ref(tor->swarm->incoming_pool) })
```

Comment:
TIL. I haven't seen `std::ref` used for a temporary array like this before :thinking: